js_modules/dagit/packages/core/src/assets/AssetNodeDefinition.tsx

```diff
@@ -65,12 +87,26 @@ export const AssetNodeDefinition: React.FC<{
               )}
             </Box>
           </Box>
-          <Box padding={{top: 16, horizontal: 24, bottom: 4}} style={{flex: 1}}>
+          <Box padding={{top: 16, horizontal: 24, bottom: 16}} style={{minHeight: 112}}>
             <Description
               description={assetNode.description || 'No description provided.'}
               maxHeight={260}
             />
           </Box>
+          {assetMetadata && (
+            <>
+              <Box
+                padding={{vertical: 16, horizontal: 24}}
+                border={{side: 'horizontal', width: 1, color: ColorsWIP.KeylineGray}}
+                flex={{justifyContent: 'space-between', gap: 8}}
```

Comment:
I think this line is probably not necessary since there's only one child here

Reply:
I looked at it again, thought you were right, and deleted it but... have a second look, there are actually two children. JSX can be hard to visually parse!